cmd/osmosisd/cmd/init.go

```diff
@@ -80,13 +82,31 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
+
+			//This is a slice of SEED nodes, not peers.  They must be configured in seed mode. 
+			//An easy way to run a lightweight seed node is to use tenderseed: github.com/binaryholdings/tenderseed
+			
+			seeds := []string {
+			"085f62d67bbf9c501e8ac84d4533440a1eef6c45@95.217.196.54:26656"} // Notional
+		 
+			
+			
+
+			
+
+			//Override default settings in config.toml
+			config.LogLevel = "error"
```

Comment:
I think the default should be `info`?

Reply:
oop, thank you!  I had been testing something.